### ISSUE TITLE
install recipes conflate "install ocaml compiler" and "install ocaml package environment"

### ISSUE THREAD

**Issue by mk270** [ocaml/v2.ocaml.org#136]
It's not clear from the Install pages that what the GODI/OPAM options install is not the same kind of thing as the OS / source options

**Comment by agarwal** [comment]
You're right. This is related to #6 and #7, but let's keep this issue open
to make sure this specific point gets addressed.
